.gitignore

```diff
@@ -14,7 +14,6 @@ gen/
 
 # Gradle files
 .gradle/
-gradle/
```

Comment:
The wrapper is *not* auto-generated, users must manually download the wrapper if they wish to build the project. If the version specified in the `gradle-wrapper.properties` file is changed, the current wrapper will update itself to the new version. Well, regardless of if this line is present in the `.gitignore`, the files are present in the repository. All this line is doing currently is preventing any updates to the Gradle Wrapper version. This change shouldn't affect anything right now, only allow for future updates of the Gradle version.